metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -908,11 +908,10 @@ class MetalsLspService(
   val isInitialized = new AtomicBoolean(false)
 
   def initialized(): Future[Unit] = {
-    loadFingerPrints()
     registerNiceToHaveFilePatterns()
-    tables.connect()
```

Comment:
Why is this line deleted?

Reply:
`loadFingerPrints()` has to do `tables.connect()` anyway. It's just a duplication really.